### PR TITLE
Fix compiler issue with ubuntu

### PR DESCRIPTION
**Pull request by dragon512**

### CODE REVIEW

**Comment by dragon512** [comment]
[approve ci autest]

File: cmd/traffic_layout/traffic_layout.cc [hunk 1]
```diff
@@ -118,35 +118,43 @@ info(int argc, const char **argv)
...
-    engine._argv.push_back(argv[i]);
-    ++i;
-  }
-  // parse the command line & put into global variable
-  if (!engine.runroot_parse()) {
-    engine.runroot_help_message(true, true);
+  try {
+    // runroot engine for operations
+    RunrootEngine engine;
+    engine._argc = argc;
```
Comment (SolidWallOfCode) [inline cmd/traffic_layout/traffic_layout.cc]: Not really related to this PR, but why does `engine` have `_argc` and a `vector` for `argv`? Isn't `_argc` just `_argv.size()`? Why have them separate? Relatedly, it seems wrong to trust `argc` for the count but not use it when populating `_argv`.

Comment (dragon512) [reply cmd/traffic_layout/traffic_layout.cc]: It was a how @chitianhao wrote it. I agree probably not needed. He can fix this in the next set of improvments

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by zwoop** [comment]
[approve ci]

**Pull request merged**